File: app/src/main/java/com/mcas2/firstdesign/FirstDesignApplication.java
package com.mcas2.firstdesign;

import android.app.Application;

import com.google.android.material.color.DynamicColors;

public class FirstDesignApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        DynamicColors.applyToActivitiesIfAvailable(this);
    }
}

File: app/src/main/java/com/mcas2/firstdesign/FirstDesignApplication.kt
package com.mcas2.firstdesign

import android.app.Application
import com.google.android.material.color.DynamicColors

class FirstDesignApplication: Application() {
    override fun onCreate() {
        super.onCreate()
        DynamicColors.applyToActivitiesIfAvailable(this)
    }
}